### ISSUE TITLE
Update the hash to curve implementation to the latest spec

### ISSUE THREAD

**Issue by arash009** [Consensys-Incorporated/teku#1927]

**Comment by ajsutton** [comment]
@benjaminion This is done now right?

**Comment by benjaminion** [comment]
Yes - it should close automatically once the `0.12.1-integration` branch is merged into master.

**Comment by benjaminion** [comment]
Didn't close automatically ¯\\\_(ツ)\_/¯
